src/middleware/model-finder.js

```diff
@@ -3,7 +3,7 @@
 import requireDirectory from 'require-directory';
 
 const whitelist = /.*?model.js$/; // Only index *model.js files
-const renamer = (name) => name.replace(/(.*?\-)model/g, 'model'); // make them all named "model"
+const renamer = (name) => name.replace(/(.*?-)model/g, 'model'); // make them all named "model"
```

Comment:
I am not sure why this change is here. Possibly it is removing an unnecessary slash?